account_invoice_validation_queued/views/queue_job_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Tecnativa - Pedro M. Baeza
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_queue_job_account_invoice_validation" model="ir.ui.view">
```

Comment:
This view is a duplicate of https://github.com/OCA/account-invoicing/commit/6ac79c298cb07f61f6f1888ef26eca702e4aa284#diff-9478f8f14832f66efde975e04566b2d4R4

Reply:
It's not the same. That view is primary and it's intended to show the specific fields for `queue.job` that are interesting to be shown.

Reply:
Does this mean that if a module has properly extended the tree view for queue_job, its changes will never appear again once this module is installed?

Reply:
Not really. This view is only intended for showing the jobs attached to an invoice, with the relevant fields in it for an "accountant". List in queue section will be the same as standard, with the same added fields.